applications/luci-app-banip/htdocs/luci-static/resources/view/banip/overview.js

```diff
@@ -564,12 +594,15 @@ return view.extend({
 		o.rmempty = true;
 
 		o = s.taboption('adv_log', form.ListValue, 'ban_loglimit', _('Log Limit'), _('Parse only the last stated number of log entries for suspicious events. To disable the log monitor at all set it to \'0\'.'));
-		o.value('0', _('0 (disable)'));
-		o.value('50', _('50'));
-		o.value('100', _('100 (default)'));
-		o.value('250', _('250'));
-		o.value('500', _('500'));
-		o.value('1000', _('1000'));
+		o.value('0');
```

Comment:
the change belongs to another PR

Reply:
I mean, maybe you just squash and merge the previous PR and then rebase the PR on top of that?
There was a lot of changes in the po files itself, I restored many translations